packages/components/src/KaizenProvider/docs/KaizenProvider.stories.mdx

```diff
@@ -0,0 +1,43 @@
+import { Meta } from '@storybook/addon-docs'
+import { CATEGORIES } from "../../../../../storybook/constants"
+
+<Meta title={`${CATEGORIES.components}/KaizenProvider`} />
+
+# KaizenProvider
+
+Provides the necessary theming context for all components, as well as facilitate the use of global UI components and translations.
+
+## Usage
+
+Ideally, the `KaizenProvider` should wrap your entire application, and that there only be one per application.
+
+```jsx
+import { KaizenProvider } from '@kaizen/components';
+
+<KaizenProvider>
+  <App />
+</KaizenProvider>
+```
+
+## Customising the theme
+
+Out of the box, the `KaizenProvider` utilises the `defaultTheme` from `@kaizen/design-tokens`. However if you want to roll your own theme, we provide a `ThemeManager` function which can apply your custom theme.
+
+```jsx
+import { ThemeManager } from '@kaizen/components'
+
+export const themeManager = new ThemeManager({
+  themeKey: "custom",
+  spacing: {
+    md: "1rem",
+  },
+});
+
+<KaizenProvider themeManager={themeManager}>
+  <App />
+</KaizenProvider>
+```
```

Comment:
Not quite sure where this belongs any more, on one hand it's the API for `KaizenProvider` on the other, it's only used for theme-ing and shouldn't be used in most cases?

Reply:
You mean the docs? Once you have i18n in the KaizenProvider, you could split it to be `KaizenProvider/Theming` and `KaizenProvider/Internationalisation`. Or you could even have the subcategory now if you'd like.